src/Uno.UI/ts/Windows/System/Profile/AnalyticsInfo.ts

```diff
@@ -0,0 +1,59 @@
+﻿namespace Windows.System.Profile {
+
+	export class AnalyticsInfo {
+		public static getDeviceType(): string {
+			var ua = navigator.userAgent;
```

Comment:
Codacy found an issue: [Identifier 'ua' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6396919)